wsmaster/che-core-api-factory-shared/src/main/java/org/eclipse/che/api/factory/shared/dto/FactoryDto.java

```diff
@@ -86,6 +86,13 @@
 
   FactoryDto withId(String id);
 
+  @FactoryParameter(obligation = OPTIONAL, setByServer = true)
+  String getSource();
```

Comment:
Could you add a few words in java docs about the purpose of this parameter? What are possible values?

Reply:
Added